pkg/cmd/operator/sidecar/sidecar.go

```diff
@@ -101,6 +105,8 @@ func NewCmd(streams genericclioptions.IOStreams) *cobra.Command {
 	cmd.Flags().StringSliceVar(&o.ExternalSeeds, "external-seeds", o.ExternalSeeds, "The external seeds to propagate to ScyllaDB binary on startup as \"seeds\" parameter of seed-provider.")
 	cmd.Flags().StringVarP(&o.NodesBroadcastAddressTypeString, "nodes-broadcast-address-type", "", o.NodesBroadcastAddressTypeString, "Address type that is broadcasted for communication with other nodes.")
 	cmd.Flags().StringVarP(&o.ClientsBroadcastAddressTypeString, "clients-broadcast-address-type", "", o.ClientsBroadcastAddressTypeString, "Address type that is broadcasted for communication with clients.")
+	cmd.Flags().StringVarP(&o.ScyllaLocalhostAddress, "scylla-localhost-address", "", "127.0.0.1", "Localhost address for connecting to ScyllaDB API (127.0.0.1 for IPv4 or ::1 for IPv6).")
+	cmd.Flags().StringVarP(&o.IPFamilyString, "ip-family", "", string(corev1.IPv4Protocol), "IP family to use for ScyllaDB (IPv4 or IPv6).")
```

Comment:
Given that we have to cross-check these values anyway, wouldn't it make sense to just leave the `ip-family` and infer the localhost address based on that? WDYT?